input/fsh/profiles_admin.fsh

```diff
@@ -20,6 +20,7 @@ Description: "Slight modification of Patient, with identifier as 0..0 and limite
 
 // We want address.country and address.postalCode to be included if available
 // See https://github.com/dvci/vaccine-credential-ig/issues/37#issuecomment-776042494
+* address MS
```

Comment:
I think our current guidance for data minimization is "name + DOB" -- it'd be good to ensure these are aligned.